test/inductor/test_cpu_repro.py

```diff
@@ -1825,6 +1825,8 @@ def run_compile_decomposed_fake_quant(
         self.assertEqual(input_grad_aten_eager, input_grad)
         self.assertEqual(input_grad_decomp_eager, input_grad)
         self.assertEqual(input_grad[1, 2, 3, 4], torch.tensor(0.0))
+        # For forward and backward kernel
+        check_metrics_vec_kernel_count(2)
```

Comment:
Is this a related change? Do we have UT on vectorized bool?

Reply:
Yean. We have a UT `python -u -m pytest -s -v inductor/test_cpu_repro.py -k test_decomposed_fake_quant_per_channel `. Previously, the forward graph has been disabled of vectorization due to the store of bool. And after this PR, we can vectorize the forward graph now.